src/interface.c

```diff
@@ -426,3 +426,13 @@ void module_register (void)
 #endif
 	plugin_register_read ("interface", interface_read);
 } /* void module_register */
+
+/*
+ * Local variables:
+ *  c-file-style: "linux"
+ *  indent-tabs-mode: t
+ *  c-indent-level: 4
+ *  c-basic-offset: 4
+ *  tab-width: 4
+ * End:
+ */
```

Comment:
I know there's prior art all over the place in the code base, and please know that this is nothing personal, but I think these mode lines are distracting at least and don't help at all. Most editors should be able to set per-project settings these days.